package.json

```diff
@@ -28,6 +28,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.10.2",
     "@ngx-translate/core": "^11.0.1",
     "@ngx-translate/http-loader": "^4.0.0",
+    "angular-cli-ghpages": "^1.0.0",
```

Comment:
This feel like it should be in the dev dependencies? Is this where `ng add` puts it?

Reply:
Yes, ng add automatically puts it in under dependencies (and not dev dependencies). Well spotted, I'll move this over to dev dependencies!